sw/device/lib/dif/dif_rstmgr.h

```diff
@@ -196,6 +210,58 @@ dif_rstmgr_result_t dif_rstmgr_reset_info_get(
 DIF_WARN_UNUSED_RESULT
 dif_rstmgr_result_t dif_rstmgr_reset_info_clear(const dif_rstmgr_t *handle);
 
+/**
+ * Enables or disables the alert crash dump capture.
+ *
+ * When enabled, will capture the alert crash dump prior to a triggered reset.
+ *
+ * @param handle A Reset Manager handle.
+ * @param state The new toggle state for the crash dump capture.
+ * @return The result of the operation.
+ */
+DIF_WARN_UNUSED_RESULT
+dif_rstmgr_result_t dif_rstmgr_alert_info_set_enabled(
```

Comment:
Should there be a `get` variant?

Reply:
Would it make sense to mention that this is reset when the system is reset (as opposed to the crash dump)? From hw spec:
> This field is automatically set to 0 upon system reset (even if rstmgr is not reset).

Reply:
Done.